src/azure-cli/azure/cli/command_modules/sql/tests/latest/test_sql_commands.py

```diff
@@ -4230,9 +4231,62 @@ def test_sql_managed_instance_mgmt(self, mi, rg):
             if expectedmessage in str(e):
                 pass
 
+        # test update managed instance to be zone redundant. Since currently there are no regions that support this, it is expected for operation to fail.
```

Comment:
Shall we add `--zone-redundant` only for `create` since `update` is supported in no region?